globals/types.ts

```diff
@@ -525,12 +539,11 @@ export type TCourseName =
   | 'Biomedical Analytics'
   | 'Compilers: Theory and Practice'
   | 'Systems Issues in Cloud Computing'
-  | 'Quantum Computing'
   | 'Introduction to Computing Law'
-  | 'Digital Health Equity'
   | 'Global Entrepreneurship'
   | 'GPU Hardware and Software'
-  | 'Security Incident Response'
```

Comment:
These three courses have been given dedicated/non-8803 course sections, corresponding updates have been made elsewhere in this PR (i.e., setting 8803's to `isDeprecated: true` and "adding" corresponding "new" sections/counterparts with dedicated/permanent course numbers